concourse/pipeline.yml

```
@@ -49,6 +69,13 @@ resources:
       aws_access_key_id: ((prod-images-aws-access-key-id))
       aws_secret_access_key: ((prod-images-aws-secret-key))
       aws_region: us-east-2
+  - name: build-and-publish-testtools-image
+    type: buildx-resource
+    source:
+      repo: ((prod-images-repo-url))/devex-dx-driver-platform-tests
```

Double-check me on this, but I think the ECR repo for the general image is `-drivers-`:
```suggestion
      repo: ((prod-images-repo-url))/devex-dx-drivers-platform-tests
```

---

It's `devex-dx-drivers-platform-tests` and `devex-dotnet-driver-platform-tests`

😿